frog/imports/ui/TeacherView/Dashboard.jsx

```diff
@@ -88,6 +88,7 @@ export class DashboardComp extends Component {
       <div style={{ width: '100%' }}>
         <aT.dashboard.Viewer
           users={users}
+          activity={this.props.activity}
```

Comment:
@lfaucon I will also need to access this data in the 'mergelog' function.

Reply:
@jkolsen could you test it now on your branch to see if it works for you?